docs/use-github-in-bcgov/transferring-repos-to-bc-gov-orgs.md

```diff
@@ -0,0 +1,102 @@
+# Transferring repositories
+
+This page describes how to transfer repositories from a team or ministry's GitHub organization to either the [bcgov](https://github.com/bcgov) or [bcgov-c](https://github.com/bcgov-c) GitHub organization.
+
+Learn more about the [bcgov](https://github.com/bcgov) and [bcgov-c](https://github.com/bcgov-c) organizations in the [B.C. Government organizations in GitHub](bc-government-organizations-in-github.md) guide.
+
+# Will it be hard?
+
```

Comment:
GitHub keeps repository transfers simple. There are a few details you’ll need to take care of for most repositories (e.g. authorizing PATs, updating team access) and some other considerations that won’t apply to everyone (e.g., packages).

The bulk of the work revolves around updating and validating any deployment pipelines that reference your repositories.

Regardless of your repository source, take care to isolate changes to your development/test deployment environment and validate before promoting to production.